builtin/logical/pki/storage.go

```diff
@@ -575,7 +575,6 @@ func fetchCertBundleByIssuerId(ctx context.Context, s logical.Storage, id issuer
 
 	var bundle certutil.CertBundle
 	bundle.Certificate = issuer.Certificate
-	bundle.IssuingCA = issuer.CAChain[0]
```

Comment:
sorry, this is used in sdk/helper/certutil/types.go, I think?  Clarification that  `pemBlock, _ = pem.Decode([]byte(c.IssuingCA))` will still work?

Reply:
Note its gated on size tho:

https://github.com/hashicorp/vault/blob/main/sdk/helper/certutil/types.go#L210-L212

So it won't really matter, we'll skip it.